integration-tests/auth_utils.go

```diff
@@ -5,26 +5,20 @@ import (
 	"encoding/json"
 	"fmt"
 	"net/http"
-	"net/url"
 
 	"code.miracl.com/mfa/pkg/gomiracl"
 	"code.miracl.com/mfa/pkg/gomiracl/bindings"
 	"code.miracl.com/mfa/pkg/gomiracl/wrap"
 )
 
-func authenticate(httpClient *http.Client, identity identity, userID string, pin int, authorizeRequestURL string) (accessResponse *accessResponse, err error) {
-	authorizeResponse, err := authorizeRequest(httpClient, authorizeRequestURL)
-	if err != nil {
-		return nil, fmt.Errorf("error making the authorize request: %w", err)
-	}
-
+func authenticate(httpClient *http.Client, identity identity, pin int, code string) error {
 	// Get pass1 proof from the token and pin (this is the U param in /pass1).
 	rand := bindings.NewRand([]byte{})
 	X := make([]byte, 32)
 	proof := make([]byte, 65)
 	xR, S, U, _, err := wrap.Client1BN254CX(int(gomiracl.SHA256), 0, identity.MPinID, rand, X, pin, identity.Token, proof)
 	if err != nil {
-		return nil, fmt.Errorf("error getting client 1: %w", err)
+		return fmt.Errorf("error getting client 1: %w", err)
 	}
 
 	// Call to /rps/v2/pass1 endpoint.
```

Comment:
The error on line 25 is not handled. I know it is not from the current pull request, it is an issue from before.

Reply:
good catch. updated!